projects/kit/components/input-phone-international/input-phone-international.options.ts

```diff
@@ -9,6 +9,7 @@ export interface TuiInputPhoneInternationalOptions {
     readonly countries: readonly TuiCountryIsoCode[];
     readonly countryIsoCode: TuiCountryIsoCode;
     readonly metadata: Observable<MetadataJson> | Promise<MetadataJson>;
+    readonly separator?: string;
 }
 
 export const TUI_INPUT_PHONE_INTERNATIONAL_DEFAULT_OPTIONS: TuiInputPhoneInternationalOptions =
```

Comment:
I think, we add `separator: '-'` to `TUI_INPUT_PHONE_INTERNATIONAL_DEFAULT_OPTIONS`

Reply:
can I add it?